<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SCreateProjectForm</class>
 <widget class="QWidget" name="SCreateProjectForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>266</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Создать проект</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="lNameForm">
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Заполните информацию о проекте</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lNameProject">
       <property name="text">
        <string>Название проекта</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lMainLang">
       <property name="text">
        <string>Основной язык</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lCompilerPath">
       <property name="text">
        <string>Транслятор</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="eNameProject"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbLang">
       <item>
        <property name="text">
         <string>LISP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Python</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>C++</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Archi Programming</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Archi Markup</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="eCompiler"/>
       </item>
       <item>
        <widget class="QPushButton" name="bChooseCompiler">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lPath">
       <property name="text">
        <string>Папка</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="ePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="bChoosePath">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>40</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
